src/validations/campaign.js

```diff
@@ -0,0 +1,19 @@
+const { Joi, validate } = require("express-validation");
+
+const createCampaign = {
+  body: Joi.object({
+    title: Joi.string().trim().required(),
+    content: Joi.string().trim().required(),
+  }),
+};
+
+const getCampaignById = {
+  params: Joi.object({
+    campaignId: Joi.string().trim().required(),
```

Comment:
Kiểm tra ObjectId
Joi.custom((item, helper) => {
        if (!ObjectId.isValid(item) || !String(new ObjectId(item)) === item)
          return helper.message('campaignId must be objectId');
        return item;
 })